core/include/bitarithm.h

```diff
@@ -164,6 +164,43 @@ static inline unsigned bitarithm_lsb(unsigned v)
 }
 #endif
 
+/**
+ * @brief   Returns the index of the first set bit in @p state, returns @p state with
```

Comment:
Maybe clarify here if "first" is the most significant or least significant bit set.

Reply:
Depends on if `clz` is available or not.

How about

> Returns the index of a set bit in @p state, returns @p state with that bit cleared.